doc/harpoon.txt

```diff
@@ -0,0 +1,281 @@
+*harpoon.txt*                                         Best plugin ever created
+
+==============================================================================
+Table of Contents                                  *harpoon-table-of-contents*
+
+1. ⇁ TOC                                                   |harpoon-⇁-toc|
+2. ⇁ The Problems                                 |harpoon-⇁-the-problems|
+3. ⇁ The Solutions                               |harpoon-⇁-the-solutions|
+4. ⇁ Installation                                 |harpoon-⇁-installation|
+5. ⇁ Getting Started                           |harpoon-⇁-getting-started|
```

Comment:
Do you think something can be done about these tags? I don't think including the symbols here is great...

Reply:
an easy solution would be to remove them from README.md

Reply:
Yeah, I wonder how prime would feel about that. Otherwise, is `panvimdoc ignore ...` allowed inline, or is that only able to ignore multiple lines? It *may* be possible to use ignore to keep the symbols in the markdown file and ignore them in the vimdoc, but I'm not sure. I'll look at the panvimdoc configs for my plugins and see if I've had to do something similar.